### PR TITLE
fix/contains list

### LINKED ISSUE #201: Using string array in filter query not working as expected
**Describe the bug**
I wanted to use **contains** filter query with a string array, but it doesn't work. 

**To Reproduce**
```
final userIds = ["a", "b"];
await client
        .from('room')
        .select()
        .contains('user_ids', userIds)

  ```

**Expected behavior**
But it works if i add closing brackets within them
```
final userIds = ["a", "b"];
await client
        .from('room')
        .select()
        .contains('user_ids', '{${userIds.map((s) => '"$s"').join(',')}}')
        ```

### PR DESCRIPTION
**Pull request by Vinzent03**
fixes supabase-community/supabase-flutter/issues/201

### CODE REVIEW

**Comment by Vinzent03** [comment]
@dshukertjr I'm confused, whether an array needs to be `{}` or `()`. I've tested all filters in the newest commit and all needed the `{}` format. Except `in`. That's the only one I've found that needs `()`.

**Comment by dshukertjr** [comment]
@Vinzent03 
Hmm, so it seems like there was a breaking change on Postgrest huh? Let me check...

**Comment by Vinzent03** [comment]
I think I would have noticed this, but may we upgrade the postgrest version in the tests?

**Review by dshukertjr — COMMENTED** [review]
@Vinzent03 
Sorry, got caught up with other things and couldn't come back to this one until now. 

I have updated the docker-compose to what is used in postgrest-js here https://github.com/supabase-community/postgrest-dart/pull/86. We could probably merge this one in and re-run the tests on this branch!

Thanks for checking different filters and figuring out which filter needs which brackets. I think the Dart library just had the wrong type of brackets until now and there was no breaking change on the Postgrest's end.

File: test/filter_test.dart [hunk 1]
```diff
@@ -71,25 +71,49 @@ void main() {
...
-    for (final item in res as List) {
-      expect((item as Map)['username'] == ('supabot'), false);
-    }
+  group("eq", () {
+    test('eq string', () async {
+      final res = await postgrest
+          .from('users')
+          .select('username')
+          .eq('username', 'supabot');
+
```
Comment (dshukertjr) [inline test/filter_test.dart:86]: Could we check that `res` is not empty for all tests that loops the res?

**Review by dshukertjr — COMMENTED** [review]
Thanks for the amazing work! I just had one little thing, but it all looks fantastic 🎉

File: test/filter_test.dart (hunk 1 above)
Comment (Vinzent03) [reply test/filter_test.dart:86]: Oh, great point!

**Review by dshukertjr — APPROVED** [review]
Thanks for the quick update! Wow, there were lot more than I thought! It looks amazing🎉

**Pull request merged**